packages/commonwealth/client/scripts/views/modals/confirm_snapshot_vote_modal.tsx

```diff
@@ -58,8 +60,9 @@ export class ConfirmSnapshotVoteModal
             <div class="vote-info-row">
               <CWText>Your voting power</CWText>
               <CWText>
-                `${formatNumberShort(totalScore)} $
-                {space.symbol.slice(0, 6).trim()}...`
+                {`${formatNumberShort(totalScore)} ${space.symbol
+                  .slice(0, 6)
```

Comment:
is this what the linter suggested? looks odd to me

Reply:
it is yeah, strange I agree